src/main/java/org/folio/rest/impl/TransactionSummaryAPI.java

```diff
@@ -1,27 +1,90 @@
 package org.folio.rest.impl;
 
+import static org.folio.rest.persist.HelperUtils.getFullTableName;
+import static org.folio.rest.persist.PostgresClient.pojo2json;
+
+import java.util.Collections;
 import java.util.Map;
 
 import javax.ws.rs.core.Response;
 
 import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Errors;
 import org.folio.rest.jaxrs.model.OrderTransactionSummary;
+import org.folio.rest.jaxrs.model.Parameter;
 import org.folio.rest.jaxrs.resource.FinanceStorageOrderTransactionSummaries;
+import org.folio.rest.persist.PgExceptionUtil;
 import org.folio.rest.persist.PgUtil;
+import org.folio.rest.persist.PostgresClient;
 
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Context;
+import io.vertx.core.Future;
 import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
 
 public class TransactionSummaryAPI implements FinanceStorageOrderTransactionSummaries {
-  static final String ORDER_TRANSACTION_SUMMARIES = "order_transaction_summaries";
+
+  private static final Logger log = LoggerFactory.getLogger(TransactionSummaryAPI.class);
+
+  public static final String ORDER_TRANSACTION_SUMMARIES = "order_transaction_summaries";
+  public static final String ORDER_TRANSACTION_SUMMARIES_LOCATION_PREFIX = "/finance-storage/order-transaction-summaries/";
+  private PostgresClient pgClient;
+  private String tenantId;
+
+  public TransactionSummaryAPI(Vertx vertx, String tenantId) {
+    this.tenantId = tenantId;
+    pgClient = PostgresClient.getInstance(vertx, tenantId);
+  }
 
   @Override
   @Validate
-  public void postFinanceStorageOrderTransactionSummaries(OrderTransactionSummary entity, Map<String, String> okapiHeaders,
+  public void postFinanceStorageOrderTransactionSummaries(OrderTransactionSummary summary, Map<String, String> okapiHeaders,
```

Comment:
Javadocs with some comments will be helpful to understand what this method is doing.